core/src/main/java/io/varhttp/BaseVarConfigurationContext.java

```diff
@@ -70,7 +71,7 @@ ControllerFactory getControllerFactory() {
 	}
 
 	@Override
-	List<Object> getDefaultFilters() {
+	LinkedList<Object> getDefaultFilters() {
```

Comment:
Why linkedlist?

Reply:
To preserve order

Reply:
I guess that was the intention, i mean :-)